docs/troubleshooting/faqs/TypeScript.mdx

```diff
@@ -52,3 +52,27 @@ You can use ESLint to enforce good uses of both ESLint and TypeScript comment di
 - The [`@eslint-community/eslint-plugin-eslint-comments`](https://eslint-community.github.io/eslint-plugin-eslint-comments) plugin can enforce general ESLint comment best practices, including requiring descriptions
 
 :::
+
+## How should I handle reports that conflict with [`verbatimModuleSyntax`](https://www.typescriptlang.org/tsconfig/#verbatimModuleSyntax)?
+
+Several TypeScript options impact how imports and exports are handled in your project, including
+
+- [`verbatimModuleSyntax`](https://www.typescriptlang.org/tsconfig/#verbatimModuleSyntax)
+- [`importsNotUsedAsValues`](https://www.typescriptlang.org/tsconfig/#importsNotUsedAsValues)
+- [`preserveValueImports`](https://www.typescriptlang.org/tsconfig/#preserveValueImports)
+- [`esModuleInterop`](https://www.typescriptlang.org/tsconfig/#esModuleInterop)
+- [`allowSyntheticDefaultImports`](https://www.typescriptlang.org/tsconfig/#allowSyntheticDefaultImports)
+
+Additionally, whether one is authoring ES Modules or CommonJs impacts `import`/`export`/`require` semantics.
+Therefore, some of our rules may not apply at all or may need special configuration in projects using these options.
+
+For example, the default behavior of [no-require-imports](/rules/no-require-imports) prohibits CommonJS `require` syntax entirely, but `verbatimModuleSyntax` requires it when authoring CommonJS modules.
+Therefore, you'll need to configure the rule to permit `import x = require('foo')` syntax.
+
+Known rules that conflict with or require special configuration to be used with `verbatimModuleSyntax` include:
+
+- [no-namespace](/rules/no-namespace#when-not-to-use-it): some reports should be ignored
+- [no-require-imports](/rules/no-require-imports#allowasimport): requires [special configuration](/rules/no-require-imports#allowasimport)
```

Comment:
Links with ambiguous text are an accessibility issue: https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html. I'm fully sighted and I find it inconvenient myself.

```suggestion
- [no-require-imports](/rules/no-require-imports): requires [configuring its `allowAsImport` option](/rules/no-require-imports#allowasimport)
```